templates/index.html

```diff
@@ -3,55 +3,165 @@
+            for the January special election and <strong>{{
+          stats.returned_special|commafy }} ballots</strong> have been returned. There were
+            <strong>{{
+          stats.returned_general|commafy }} ballots </strong> returned early in the general
+            election, so we are only
+            <strong>{{ (100.0 * stats.returned_special / stats.returned_general)|round(1) }}%</strong>
+            of the way there
+            <em>(as of {{ stats.update_time.strftime('%B %-d at %-I:%M %p EST') }})</em>.
+          </p>
       </div>
     </div>
-    <div class="form-row">
-      <div class="col-md-12 col-sm-12 mb-3">
-        Optional fields:
-      </div>
+
+    <div>
+      <h3 class='text-center'>Do your part: Help your friends in Georgia vote</h3>
+      <hr class='my-5'>
     </div>
-    <div class="form-row">
-      <div class="col-md-4 col-sm-12 mb-3">
-        <label for="input-first">Middle name/initial</label>
-        <input type="text" class="form-control" placeholder="Middle name" name="middle" id="input-middle">
+
+    <div>
+      <!-- TODO: Have the # be its own column and everything else to the right for a bit more separation? Depends if we are cool with the wider container though. -->
+      <h4>1️⃣ Who do you know in Georgia?</h4>
+      <p>Check which of your Facebook friends lives in Georgia.</p>
+      <a href="https://www.facebook.com/search/top?q=my%20friends%20who%20live%20in%20georgia"
+        target="_blank" rel="noopener" class="btn text-white mb-3"
+        style="background-color: #e12814;">Find friends in Georgia on Facebook</a>
+      <p><em>Did you know it's possible to find your friends on Facebook with a text search? Click
+          the button above to run the search, or just go to Facebook and type "my friends in
+          Georgia" into the search box.</em></p>
+      <hr class="my-5">
+    </div>
+
+    <div class="mt-4">
+      <h4>2️⃣ Let them know there's an election</h4>
+      <p>In most elections, over 30% of voters don't know when the election is. Make sure your
+        friends know that the election is on <strong>Tuesday, January 5, 2021</strong>. Anyone can
+        vote early in-person starting on <strong>Monday, December 14, 2020</strong>.</p>
+
+      <div class="d-flex flex-column justify-content-end align-items-end">
+        <div class="card bg-primary rounded-lg text-white mb-2" style="width: 18rem;">
+          <div class="card-body">
+            <p class="card-text">
+              Hi Emily! I'm sure you already know that the Georgia runoff election, the one that
+              determines who controls the Senate, is on January 5th. I just wanted to pass along
+              this link to find your early voting location -- you can start voting today!<br><br>
+              <strong>www.voteamerica.com/where-to-vote</strong></p>
+          </div>
+        </div>
+
+        <!-- TODO: See if this works, I have an Android rn -->
```

Comment:
Could one of you test the link e2e? I have a Pixel so am using Google messages on the web